presale/views/sale_order_views.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version='1.0' encoding='utf-8'?>
+<odoo>
+    <record id="sale_order_presale_order" model="ir.ui.view">
+        <field name="name">sale.order.form.inherit</field>
+        <field name="model">sale.order</field>
+        <field name="inherit_id" ref="sale.view_order_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//group[@name='order_details']" position="inside">
+                <field name='presale_order_id' groups="presale.group_presale_users"/>
+            </xpath>
+            <xpath expr="//button[@name='action_quotation_send'][4]" position="attributes">
```

Comment:
`[4]` might be unreliable as other modules could override the same view and add/remove buttons with the same name; you can use `[last()]` to target the last node matching the expression